netdata/netdata
# Update binaries (eBPF)

##### Summary
Update all binaries used with `eBPF.plugin` given conditions to show process and thread name on functions without necessity to open `/proc` files.

##### Test Plan

1. Edit your `/etc/netdata/ebpf.d.conf` and set:
```conf
[ebpf programs]
    cachestat = yes
    dcstat = yes
    fd = yes
    oomkill = yes
    process = yes
    shm = yes
    socket = yes
    swap = yes
    vfs = yes
```
2. Start netdata.
3. Take a look if you do not have `libbpf` errors after few seconds. They appear during load time when there is an issue.
```sh
$ grep libbpf /var/log/netdata/*
```

##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
